sdk/test-utils/recorder/src/options.ts

```diff
@@ -0,0 +1,31 @@
+import { createHttpHeaders, createPipelineRequest, HttpClient } from "@azure/core-rest-pipeline";
+import { paths } from "./utils/paths";
+import { RecorderError } from "./utils/utils";
+
+export type RecordingOptions = {
+  HandleRedirects?: boolean;
```

Comment:
```suggestion
  handleRedirects?: boolean;
```
?

Reply:
Done, I was initially just matching the casing on the test proxy side. Added a mapping